plugins/package-managers/node/src/funTest/assets/projects/synthetic/pnpm/project-with-lockfile-expected-output.yml

```diff
@@ -47,7 +47,7 @@ project:
         - id: "NPM::domhandler:2.4.2"
           dependencies:
           - id: "NPM::domelementtype:1.3.1"
-        - id: "NPM::domutils:1.5.1"
+        - id: "NPM::domutils:1.7.0"
```

Comment:
> The cyclic reference from the workspace root project to itself is now included.

Maybe a bit off-topic, why why is there such a cyclic dependency to begin with? It sounds somewhat wrong to have it at all.

Reply:
I investigated this a bit by removing this circular dependency from the `package.json`.
As a result the `old` implementation did not find any dependencies. Maybe this was a workaround to
make the old `pnpm` implementation "work" with the workspace. If this would really be the case, then it would
mean that nobody is using the pnpm workspaces feature, e.g. would have expected a bug report.

I deliberately didn't remove the cyclic dependency in this PR, as I wanted to make this transparent.